src/utils.js

```diff
@@ -0,0 +1,9 @@
+export const getRatingColor = (rating) => {
+  const normalizedRating = Math.min(1, Math.max(0, (rating - 5) / 5));
+
+  //  red and green values based on the normalized rating
+  const red = Math.min(255, Math.max(0, 255 - normalizedRating * 255));
```

Comment:
Is  `255 - normalizedRating * 255` 
the same as saying: `255 - (normalizedRating * 255)` ?

Can you verify?

Reply:
yes!